GitUI/TranslatedStrings.cs

```diff
@@ -84,12 +84,13 @@ internal sealed class TranslatedStrings : Translate
 
 - For a single selected commit, show the difference with its parent commit.
 - For a single selected merge commit, show the difference with all parents.
-- For two selected commits with a common ancestor (BASE):
-   - Show the difference between the commits.
-   - The difference of unique files from BASE to each of the selected commits.
-   - The difference of common files (identical changes) from BASE to the commits.
-- For multiple selected commits (up to four), show the difference for all the first selected with the last selected commit.
-- For more than four selected commits, show the difference from the first to the last selected commit.");
+- For two selected commits with a common ancestor (BASE), show the difference
+between the commits as well as the difference from BASE to the selected commits.
```

Comment:
Do we really need to wrap the lines?

Reply:
The existing tooltip was too wide already for lower resolution screens.
I find it too wide also on my second screen, 1600x1200 and unusable on my old laptop (1280x720?).